packages/v4/src/content/design-guidelines/content/brand-voice-and-tone.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Brand voice and tone
+section: content
+hideTOC: true
+showTitle: true
+---
+
+In a business context, brand is the identity of a company that people recognize based on an emotional and psychological connection as well as factual information.
+
+All your UX copy should align with your company’s brand, contributing to a consistent and authentic experience across every content channel users interact with.
+To keep your UX copy on-brand, consider style, voice, and tone:
+
+- **Style** is the way you use grammar, punctuation, and syntax. It’s influenced by how your words work together and what effect your writing has on the user. 
+
+- **Voice** is the personality that’s reflected in your writing.
+
+- **Tone** is the approach you take that’s appropriate for the context of your content. Tone encompasses the user’s emotions that need to be accounted for and the resulting approach you must take.
+
+Let’s consider how we use the Red Hat brand voice in our UX copy. If Red Hat were a person, they would be:
+
+- Helpful but humble (not arrogant).
+- Authentic but adaptable (not stubborn).
+- Open but ordered (not chaotic).
+- Brave but balanced (not reckless).
+
+## Voice traits
+To reflect the Red Hat voice in UX copy for Red Hat products, we have UX voice traits. Each voice trait is an extension of the Red Hat voice.
+
+<table align="center" style="margin: 0px auto; table-layout:fixed;">
```

Comment:
I see you are center aligning this on the page, but I think it looks kind of odd that way as the page resizes. Think it would make more sense to left-align. Also, maybe bold the column headings?

Reply:
Good point. I'll do that!